edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/RequestedEquipmentTO.java

```diff
@@ -22,7 +28,7 @@ public record RequestedEquipmentTO(
   @NotNull
   Float units,
 
-  List<@NotBlank @Size(max = 15) String> equipmentReferences,
+  List<@NotBlank @Size(max = 15) @ISO6346EquipmentReference String> equipmentReferences,
```

Comment:
You might as well change size to be `min=11, max=11` here as `@ISO6346EquipmentReference` has those size restrictions (or remove `@Size` and let `@ISO6346EquipmentReference` deal with it)`.